### PR TITLE
readds an accidentally removed feature

### PR DESCRIPTION
**Pull request by zeskorion**
## About The Pull Request
re-adds anomaly core activation effects. These were added in #1218, but were later removed, i assume by accident, by #1623

## Why It's Good For The Game
readds a missing feature, noticed it was missing when checking the code for it

## Changelog
:cl:
fix: anomaly cores now have assembly activation effects again
/:cl:

### CODE REVIEW

**Comment by Tsar-Salat** [comment]
Ah, I assumed it was simply nerfed or something.

Anomaly core in bomb assembly just reform as anomalies yes?

File: code/modules/assembly/signaler.dm [hunk 1]
```diff
@@ -199,6 +199,15 @@
 		A.anomalyNeutralize()
 	return TRUE
 
+/obj/item/assembly/signaler/anomaly/activate()
+	if(!..())//cooldown processing
+		return FALSE
+	var/type = src.anomaly_type
+	var/obj/effect/anomaly/B = new type(get_turf(src))
+	B.anomalyEffect()
+	B.Destroy()
```
Comment (Kapu1178) [inline code/modules/assembly/signaler.dm:208]: ???????????????????????

Manual destroy call??

**Review by Kapu1178 — CHANGES_REQUESTED** [review]

File: code/modules/assembly/signaler.dm [hunk 2]
```diff
@@ -199,6 +199,15 @@
 		A.anomalyNeutralize()
 	return TRUE
 
+/obj/item/assembly/signaler/anomaly/activate()
+	if(!..())//cooldown processing
+		return FALSE
+	var/type = src.anomaly_type
```
Comment (Archanial) [inline code/modules/assembly/signaler.dm:205]: Why do you need src here?

File: code/modules/assembly/signaler.dm (hunk 1 above)
Comment (Archanial) [reply code/modules/assembly/signaler.dm:208]: Wtf

File: code/modules/assembly/signaler.dm [hunk 3]
```diff
@@ -199,6 +199,15 @@
...
+	var/obj/effect/anomaly/B = new type(get_turf(src))
+	B.anomalyEffect()
```
Comment (PowerfulBacon) [inline code/modules/assembly/signaler.dm:207]: This requires the deltaTime parameter

**Comment by zeskorion** [comment]
Bleh, i just copypasted the code from the original pr,will look later

**Comment by PowerfulBacon** [comment]
> Bleh, i just copypasted the code from the original pr,will look later

Original code is no longer compatible due to deltaTime changes

**Review by PowerfulBacon — CHANGES_REQUESTED** [review]
Address reviews

**Pull request closed without merging**